docs/user-guide/getting-started.md

```diff
@@ -23,14 +23,14 @@ yarn add eslint --dev
 You should then set up a configuration file, and the easiest way to do that is to use the `--init` flag:
```

Comment:
Should we now remove the mention of "--init flag" from this sentence?

Reply:
removed in https://github.com/eslint/eslint/pull/15150/commits/7b932c4e332cae9a744f3983493b4d8d7eef6de3